.circleci/.env.test

```diff
@@ -0,0 +1,11 @@
+# For a complete list of core environment variables see
+# https://docs.silverstripe.org/en/4/getting_started/environment_management/#core-environment-variables
+
+# DB credentials
+SS_DATABASE_CLASS="MySQLDatabase"
```

Comment:
No need to declare the database class, it should pick up manually.
Nothing wrong with it though, just saying :)

Reply:
I thought it defaults to `MySQLPDODatabase`. As I've heard it is slower and has issues.